File: backend/src/main/java/com/demo/student_management/dto/admin/GiaoVienResponse.java
package com.demo.student_management.dto.admin;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class GiaoVienResponse {
    private String idGiaoVien;
    private String tenGiaoVien;
    private String tenDangNhap;
    private String sdt;
    private String email;
    private String gioiTinh;
    private String idLop;
    private String tenLop;
    private String idLopChuNhiem;
    private String tenLopChuNhiem;
    private String idMonHoc;
    private String tenMonHoc;
    private Boolean active;
}

File: frontend/StudentManagement.Shared/Dtos/Admin/GiaoVienResponse.cs
namespace StudentManagement.Shared.Dtos.Admin;

public sealed class GiaoVienResponse
{
    public string IdGiaoVien { get; set; } = string.Empty;
    public string TenGiaoVien { get; set; } = string.Empty;
    public string TenDangNhap { get; set; } = string.Empty;
    public string Sdt { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string GioiTinh { get; set; } = string.Empty;
    public string IdLop { get; set; } = string.Empty;
    public string TenLop { get; set; } = string.Empty;
    public string IdLopChuNhiem { get; set; } = string.Empty;
    public string TenLopChuNhiem { get; set; } = string.Empty;
    public string IdMonHoc { get; set; } = string.Empty;
    public string TenMonHoc { get; set; } = string.Empty;
    public bool Active { get; set; } = true;
}
